src/wagtail_ai/models.py

```diff
@@ -0,0 +1,106 @@
+import uuid
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from wagtail.search import index
+
+DEFAULT_PROMPTS = [
```

Comment:
**Suggestion:** Can we set an explicit type for this? We then make sure we're settings things correctly and that typing is used in other places that use this list.

Otherwise, the type checker has to infer the type and that may mean we introduce more room for error as it interprets what's inside the list (even if it includes a human error), not what should be in the list.

I think the type inferred would be something like `dict[str, str]` that allows any keys, rather than a specific TypedDict that only permits specific keys in the mapping.

```python

class PromptDict(TypedDict):
  ....

DEFAULT_PROMPTS: Sequence[PromptDict] = [
...
```